R/estimate_outcomes.R

```diff
@@ -12,8 +12,8 @@
 #' @return A `<data.frame>` with the columns in `data`, and with two additional
 #' columns:
 #'
-#'  - `"known_outcomes"` for the total number of known outcomes on that day
-#' of the outbreak, and
+#'  - `"estimated_outcomes"` for the total number of outcomes estimated to be
+#' known on that day of the outbreak, and
 #'
 #'  - `"u_t"` for the under-reporting factor.
```

Comment:
Here it'd be useful to write down what "u_t" represents, is it the % of cases that have been ascertained at the specific points of the outbreak?

Reply:
Thanks, does "`"u_t"` for the ratio of cumulative number of estimated known outcomes and the cumulative number of cases reported until each date specified in `data`" work?